spec/models/preserved_copy_spec.rb

```diff
@@ -309,4 +314,53 @@
       end
     end
   end
+
+  describe '#create_archive_preserved_copies!' do
```

Comment:
one thing i'd considered was writing tests for `#create_archive_preserved_copies!` that used two different druids, similarly to what i did for the `which_have_archive_copy` and `which_need_archive_copy` scopes on `ArchiveEndpoint`.  i'd still be open to do that, but felt like this was complete enough to be a PR as-is, especially since the scope that `#create_archive_preserved_copies!` relies on to figure out what to create is tested in that way.  figured it might be better to get this merged and ticket follow-on work.

Reply:
I agree it would be a good idea to have tests for `create_archive_preserved_copies!` using 2 different druids, and also agree that's fine for follow-on work. Otherwise LGTM.

Reply:
#986